src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -324,6 +324,31 @@ class HttpGoogleServicesDAO(
     }
   }
 
+  override def getBucketUsage(bucketName: String, maxResults: Option[Long]): Future[BigInt] = {
+    def sumItemSizes(items: java.util.List[StorageObject]) = {
+      (BigInt(0) /: items) { (sum, o) => sum + o.getSize }
```

Comment:
According to the book I was looking at, `/:` is a `foldLeft` operator.

There's some extra complexity that `getSize` returns a `java.math.BigInteger`. By being a little more verbose and aggressively moving to `BigInt`s as soon as possible, `reduce(_ + _)` can actually become `sum`.

    items.map((s: StorageObject) => BigInt(s.getSize)).sum